alot/account.py

```diff
@@ -214,6 +214,9 @@ def errb(failure):
             logging.error(failure.value.stderr)
             raise SendingMailFailed(errmsg)
 
+        # make sure self.mail is a string
+        mail = str(mail)
```

Comment:
I'm not sure how this is worse than [what we did before](https://github.com/pazz/alot/pull/1115/commits/f67d3ac2bca0dd0dda6d5998ed1131d5b383f995#diff-feec6663d333aaa89ed596841b6acc19L242).
`mail` is, as assumed in the function docstring, either a `str` already, in which this does nothing. Or it is an `email.message`, in which case the email lib should do the right thing..